.travis.yml

```diff
@@ -7,7 +7,7 @@ env:
   global:
     - REACT_ENV=development
     - REACT_APP_LOG_LEVEL=debug
-    - REACT_APP_NETWORK_ID=4447
+#    - REACT_APP_NETWORK_ID=4447
```

Comment:
where are we getting this value if this commented out?

Reply:
From this default value:

```javascript
"networkId": process.env.REACT_APP_NETWORK_ID || 42,
```

https://github.com/GoodDollar/GoodDAPP/blob/165854539-increase-test-coverage/src/config/config.js#L6